### ISSUE TITLE
The submitted draft is too difficult to reach

### ISSUE THREAD

**Issue by MoritzLost** [engram-design/Workflow#167]
### Describe the bug

We're trying the update to 2.0.0 right now, and everything looks fine. But one thing we noticed is that as a publisher, it's pretty cumbersome to view the draft being submitted. Previously, the submission link in the workflow overview took you directly to the draft being edited. Now this link leads to the overview for the submission, which is fine. However, both the link in the _Entry_ column in the overview and the link to the entry on the submission page lead to the _canonical_ version of the entry, not the draft the submission belongs to. So the only way to review the draft the submission belongs to is to open the entry and then manually select the draft from the revisions dropdown. In addition, the submission overview doesn't indicate which draft a submission belongs to, so if there are multiple draft, the publisher has no way to tell which draft is being submitted.

I propose some changes to amend this:

1. The link in _Entry_ column in the workflows overview should lead directly to the draft the submission belongs to, not to the canonical version of the entry.
2. Same goes for the _Entry_ link in the sidebar of the submission details view.
3. The submission details view should clearly indicate which draft the submission belongs to (and link to that draft).

### Steps to reproduce

1. As an editor, create multiple drafts and submit one of them.
2. As a publisher, go to the submission overview page and click the entry link for that submission. This will lead to the canonical version of the entry. The draft being submitted has to be manually opened through the revisions dropdown, and since there are multiple drafts, the publisher has no indication which draft the submission belongs to.

### Craft CMS version

4.3.3

### Plugin version

2.0.0

### Multi-site?

No

**Comment by engram-design** [comment]
Of course, makes total sense. So this is only an issue when making submissions on already-published drafts. Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by MoritzLost** [comment]
@engram-design Thanks for the quick fix! I just tried the new dev version, works perfectly 👍

**Comment by engram-design** [comment]
Fixed in [2.0.2](https://github.com/verbb/workflow/releases/tag/2.0.2)
